app/controllers/auth/WithAuthorisedForTai.scala

```diff
@@ -132,4 +133,6 @@ case class TaiUser(authContext: AuthContext, person: Person){
 
     def getUTR = authContext.principal.accounts.sa.map(sa => sa.utr.utr).getOrElse("")
 
+    def nino: Nino = Nino(getNino)
```

Comment:
Are you planning on changing all of the calls of user.getNino to user.nino

Reply:
No I wasn't at first, do you think we should deprecate the `getNino` api?